types/underscore/underscore-tests.ts

```diff
@@ -662,6 +663,139 @@ interface ChainTypeExtractor {
 
 declare const extractChainTypes: ChainTypeExtractor;
 
+// Types
+
+// Iteratee
+{
+    // functions
+    const listFunctionIteratee: _.Iteratee<_.List<StringRecord>, string> = (element, key, collection) => {
+        element; // $ExpectType StringRecord
+        key; // $ExpectType number
+        collection; // $ExpectType List<StringRecord>
+        return element.a;
+    };
+    listFunctionIteratee(stringRecordList[0], 0, stringRecordList);
```

Comment:
Are you only interested in whether this expression passes the typechecks or did you forget an `$ExpectType`?

Reply:
Totally forgot an `$ExpectType`, thanks! Not sure how I managed that since I thought I copy/pasted these.